docs/src/main/sphinx/functions/table.rst

```diff
@@ -47,6 +47,58 @@ by the specified catalog, the query fails.
 The table function name is resolved case-insensitive, analogically to scalar
 function and table resolution in Trino.
 
+Arguments
+^^^^^^^^^
+
+There are three types of arguments.
+
+1. Scalar arguments
+
+They must be constant expressions, and they can be of any SQL type, which is
+compatible with the declared argument type::
+
+    factor => 42
+
+2. Descriptor arguments
+
+Descriptors consist of fields with names and optional data types::
+
+    schema => DESCRIPTOR(id BIGINT, name VARCHAR)
+    columns => DESCRIPTOR(date, status, comment)
+
+To pass ``null`` for descriptor, use::
+
+    schema => CAST(null AS DESCRIPTOR)
+
+3. Table arguments
+
+You can pass a table name, or a query. Use the keyword ``TABLE``::
+
+    input => TABLE(orders)
+    data => TABLE(SELECT * FROM region, nation WHERE region.regionkey = nation.regionkey)
```

Comment:
what the context of that query in terms of catalog, schema..

Reply:
It's the context of the enclosing query. I don't think it needs a comment.

This is not like a query pass-through. In query pass-through, you pass a textual query. Here, you pass the actual data. The query is executed by the engine, and the result is streamed into the TF operator.

Reply:
ok ..